consensus/fork/galactica.go

```diff
@@ -164,13 +164,13 @@ func CalculateReward(gasUsed uint64, rewardGasPrice, rewardRatio *big.Int, isGal
 	return reward
 }
 
-func ValidateGalacticaTxFee(tr *tx.Transaction, baseFee, baseGasPrice *big.Int) error {
-	galacticaItems := GalacticaTxGasPriceAdapter(tr, baseGasPrice)
-	if galacticaItems.MaxFee.Cmp(baseFee) < 0 {
-		if tr.Type() == tx.TypeLegacy {
-			return errBaseFeeTooHighForLegacyTx
-		}
-		return errBaseFeeTooHighForDynamicFeeTx
+func ValidateGalacticaTxFee(tr *tx.Transaction, baseGasPrice, legacyTxDefaultGasPrice *big.Int) error {
+	// pow is not accounted for verifying if gas is enough to cover block base fee
+	feeItems := GalacticaTxGasPriceAdapter(tr, tr.GasPrice(legacyTxDefaultGasPrice))
+
+	// do not accept txs with less than the block base fee
+	if feeItems.MaxFee.Cmp(baseGasPrice) < 0 {
```

Comment:
Will look into it more later but by removing `errBaseFeeTooHighForDynamicFeeTx` we are essentially back to same problem we discussed yesterday: if the user submits a transaction, there is no `max fee per gas` so the user does not know what to do since there is no attribute with that name.

What was merged yesterday, stating clearly the problem and providing options to resolve it is much more clear IMO.

Reply:
Resolved offline - updated error naming.